documentation/examples/contrib/kubernetes/rabbitmq/README.md

```diff
@@ -0,0 +1,28 @@
+# RabbitMQ Scraping
+
+This is an example on how to setup RabbitMQ so Prometheus can scrap data from it.
+It uses a third party [RabbitMQ exporter](https://github.com/kbudde/rabbitmq_exporter).
+
+Since the [RabbitMQ exporter](https://github.com/kbudde/rabbitmq_exporter) needs to
+connect on RabbitMQ management API to scrap data, and it defaults to localhost, it is
+easier to simply embed the **kbudde/rabbitmq-exporter** on the same pod as RabbitMQ,
+this way they share the same network.
+
+With this pod running you will have the exporter scraping data, but Prometheus have not
+yet found the exporter and is not scraping data from it.
+
+For more details on how to use kubernetes service discovery take a look on the 
+[documentation](http://prometheus.io/docs/operating/configuration/#kubernetes-sd-configurations-kubernetes_sd_config)
+and on the [available examples](./documentation/examples).
+
+After you got Kubernetes service discovery up and running you just need to advertise that RabbitMQ
+is exposing metrics. To do that you need to define a service that:
+
+* Exposes the exporter port
+* Add the annotation: prometheus.io/scrape: "true"
```

Comment:
The structure here doesn't fit to "define a service that:". How about "Has the annotation: ..."? Same below.